src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  --side: 15px;
+  height: var(--side);
+  width: var(--side);
+  margin-right: 4px;
+  background: url(images/star.svg) center no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(1n) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
this are the same rules, write it like that:

```
 .stars--4 .stars__star:nth-child(-n+),
 .stars--5 .stars__star:nth-child(-n+) {
  background-image: url(images/star-active.svg);
}
```

Reply:
fixed